security/Makefile

```diff
@@ -23,8 +23,6 @@ check.prereqs:
 	../bin/verify_go_version.sh
 
 clean:
-	rm -f docker/istio_ca
```

Comment:
I'm not clear on the motivation for this.  These are deleting the go executables that get copied over from go/bin as part of making a docker image and are unrelated to the .crt/.key files.

Reply:
You are right. Will remove this from the PR.